sphinx_ansible_theme/static/css/ansible.css

```diff
@@ -484,3 +484,5 @@ tr .ansibleOptionLink {
 .rst-content div[class^="highlight"] pre {
     white-space: pre-wrap;
 }
+
+.rst-content dt { margin-bottom: 0; }
```

Comment:
Did you test this? I don't think this will be used since the selectors which add the margin are more specific than this one (`.rst-content dl dt`).

Reply:
I just tested online with firefox. I'm trying to find how to build it meanwhile.

Reply:
```suggestion
.rst-content dl dt {
    margin-bottom: 0;
}
```

Reply:
Thanks !